packages/web/src/views/launchpad/components/launchpad-active-projects/launchpad-active-projects-content/launchpad-active-projects-card-list/LaunchpadActiveProjectsCardList.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import { LaunchpadProjectResponse } from "@repositories/launchpad/response";
+
+import LaunchpadActiveProjectCard from "./launchpad-active-project-card/LaunchpadActiveProjectCard";
+import {
+  ActiveProjectsCardListWrapper,
+  ActiveProjectsGridWrapper,
+} from "./LaunchpadActiveProjectsCardList.styles";
+import LoadMoreButton from "@components/common/load-more-button/LoadMoreButton";
+
+interface LaunchpadActiveProjectsCardListProps {
+  activeProjectList: LaunchpadProjectResponse[];
+  showLoadMore: boolean;
+  loadMore: boolean;
+
+  onClickLoadMore: () => void;
+  moveProjectDetail: (poolId: string) => void;
+}
+
+const LaunchpadActiveProjectsCardList: React.FC<
+  LaunchpadActiveProjectsCardListProps
+> = ({
+  activeProjectList,
+  showLoadMore,
+  loadMore,
+  onClickLoadMore,
+  moveProjectDetail,
+}) => {
+  return (
+    <ActiveProjectsCardListWrapper>
+      <ActiveProjectsGridWrapper>
+        {activeProjectList && activeProjectList.length > 0 && (
```

Comment:
What does this condition mean?
From my point of view, the processing is missing when there is no project.